pkg/process/checks/process.go

```diff
@@ -367,3 +389,21 @@ func (p *ProcessCheck) createTimesforPIDs(pids []int32) map[int32]int64 {
 	}
 	return createTimeForPID
 }
+
+// mergeProcWithSysprobeStats takes a process by PID map and fill the stats from system probe into the processes in the map
+func mergeProcWithSysprobeStats(procs map[int32]*procutil.Process, pu *net.RemoteSysProbeUtil) {
+	pStats, err := pu.GetProcStats()
+	if err == nil {
+		for pid, proc := range procs {
+			if s, ok := pStats.StatsByPID[pid]; ok {
+				proc.Stats.OpenFdCount = s.OpenFDCount
+				proc.Stats.IOStat.ReadCount = s.ReadCount
+				proc.Stats.IOStat.WriteCount = s.WriteCount
+				proc.Stats.IOStat.ReadBytes = s.ReadBytes
+				proc.Stats.IOStat.WriteBytes = s.WriteBytes
+			}
+		}
+	} else {
+		log.Debugf("cannot do GetProcStats for process check: %s", err)
```

Comment:
nit: should we mention sysprobe in the logs to make it easier to debug?

Reply:
Sure.